vendor/plugins/edu.harvard.dce.paella.timedCommentsPlugin/timedCommentsOverlay.js

```diff
@@ -73,10 +74,15 @@ Class ("paella.plugins.TimedCommentsOverlay", paella.EventDrivenPlugin, {
     paella.events.hideTimedComments = "dce:hideTimedComments";
     return[paella.events.showTimedComments, paella.events.hideTimedComments, paella.events.refreshTimedComments, paella.events.play, paella.events.timeupdate, paella.events.pause, paella.events.endVideo];
   },
-
+  init: function() {
```

Comment:
What if config.adminRoles is empty? Don't we want to use the default?

Reply:
It's more flexible when the user (via config) has the option to remove the default role all together to give the same UI view to all users. Currently, the default is only used if the adminRoles param is missing all together. But, if it's there and empty, the special admin UI "look" is <intentionally> omitted for users with admin role.

Reply:
@rute-santos  Have Tested the update with empty/missing user roles, roles as string vs array, missing adminRoles config (uses default admin role), empty adminRoles config (everyone's a user), adminRoles config without the default role (i.e. just the special dce admin role), default admins are treated as users and only users with special admins roles are admins.

Reply:
So, in English, if config.adminRoles exists but is intentionally empty, everyone is seen as a user and gets the user view. 

If the config.adminRoles param does not exist, the plugin uses ROLE_ADMIN which gives backwards compatibility with 1x.

For this pull, config.adminRoles does exist and contains the ROLE_ADMIN along with all other ADMIN roles that are site configured within the site configured back-end annot filter.